website/docs/r/schema_template_service_graph.html.markdown

```diff
@@ -33,16 +30,18 @@ resource "mso_schema_template_service_graph" "test_sg" {
 * `schema_id` - (Required) Schema ID where Service Graph to be created.
 * `template_name` - (Required) Template Name where Service Graph to be created.
 * `service_graph_name` - (Required) Name of Service Graph.
```

Comment:
should we keep this consistent with website/docs/r/schema_site_service_graph.html.markdown?

Reply:
rephrased.